bundles/org.openhab.binding.teleinfo/src/main/resources/ESH-INF/thing/common-channel-types.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="teleinfo"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<channel-type id="isouscType">
+		<item-type>Number:ElectricCurrent</item-type>
+		<label>ISOUSC</label>
+		<description>ISOUSC</description>
+		<category>Energy</category>
+		<state pattern="%d A" readOnly="true" />
+	</channel-type>
+
+	<channel-type id="ptecType">
+		<item-type>String</item-type>
+		<label>PTEC</label>
+		<description>PTEC</description>
+		<category>Energy</category>
+		<state pattern="%s" readOnly="true">
+			<options>
+				<option value="TH">TH</option>
+				<option value="HC">HC</option>
+				<option value="HP">HP</option>
+				<option value="HN">HN</option>
+				<option value="PM">PM</option>
+				<option value="HCJB">HCJB</option>
+				<option value="HCJW">HCJW</option>
+				<option value="HCJR">HCJR</option>
+				<option value="HPJB">HPJB</option>
+				<option value="HPJW">HPJW</option>
+				<option value="HPJR">HPJR</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="hhphcType">
+		<item-type>String</item-type>
+		<label>HHPHC</label>
+		<description>HHPHC</description>
+		<category>Energy</category>
+		<state pattern="%s" readOnly="true">
+			<options>
+				<option value="A">A</option>
+				<option value="C">C</option>
+				<option value="D">D</option>
+				<option value="E">E</option>
+				<option value="Y">Y</option>
+			</options>
+		</state>
+	</channel-type>
+
+	<channel-type id="lastUpdateType">
+		<item-type>DateTime</item-type>
+		<label>Last update</label>
```

Comment:
Words in labels should be capitalized. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions

Reply:
Ok